meson.build

```diff
@@ -7,7 +7,7 @@ project('WPEBackend-fdo', ['c', 'cpp'],
 		'cpp_std=c++11',
 	],
 	license: 'BSD-2-Clause',
-	version: '1.7.1',
+	version: '1.9.1',
```

Comment:
Same here, it should be `1.9.0` and the libtool versioning numbers below need to be the same as in `CMakeLists.txt`.